components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java

```diff
@@ -2239,4 +2258,501 @@ public static ClusterBean getClusterInfo(String clusterId) throws RestAPIExcepti
 
         return ObjectConverter.convertClusterToClusterBean(cluster, clusterId);
     }
+
+    //util methods for Tenants
+
+    /**
+     * Add Tenant
+     *
+     * @param tenantInfoBean
+     * @throws RestAPIException
+     */
+    public static void addTenant(org.apache.stratos.common.beans.TenantInfoBean tenantInfoBean) throws RestAPIException {
+
+        try {
+            CommonUtil.validateEmail(tenantInfoBean.getEmail());
+        } catch (Exception e) {
```

Comment:
It's generally not acceptable to catch generic exceptions. If done intentionally, might be good to add a comment explaining the reason why it was done this way.